src/System.Linq.Expressions/tests/InterpreterTests.cs

```diff
@@ -17,7 +17,8 @@ public static class InterpreterTests
         [Fact]
         public static void VerifyInstructions_Simple()
         {
-            Expression<Func<string, bool>> f = s => s != null && s.Substring(1).Length * 2 > 0;
+            // Using an unchecked multiplication to ensure that a mul instruction is emitted (and not mul.ovf)
+            Expression<Func<string, bool>> f = s => s != null && unchecked(s.Substring(1).Length * 2) > 0;
```

Comment:
I can change or remove it but I believe it to be correct. ```VerifyInstructions(LambdaExpression, string)``` calls ```GetInstructions(LambdaExpression)``` which compiles the expression. The latter's output is then used as the actual test result.